iron-ajax.html

```diff
@@ -317,9 +317,11 @@
      * @return {Object}
      */
     get requestHeaders() {
-      var headers = {
-        'Content-Type': this.contentType
-      };
+      var headers = {};
+
+      if ( this.contentType !== 'false' ) {
```

Comment:
Perhaps we should check for headers keys whose values `== null`, and not set those ones..